infra/src/main/kotlin/fi/fta/geoviite/infra/configuration/RequestFilter.kt

```diff
@@ -154,6 +183,11 @@ class RequestFilter @Autowired constructor(
             )
         } else if (request.requestURI == "/actuator/health" && headers.none { h -> h.startsWith("x-iam") }) {
             healthCheckUser
+        } else if (
+            environment.activeProfiles.contains("integration-api") &&
+            request.requestURI.startsWith("/rata-vkm")
```

Comment:
API-käyttäjää yritetään määrittää vain kun tuo profiili on käytössä, sekä lisärajauksena laitettu toistaiseksi tuo rata-vkm:n juuripolku.

Teoriassa jos tuota juuripolkua rata-vkm -rajapintaan ei olisi ja springi startattaisiin backend- JA integration-api profiililla (ei-lokaalisti), niin kävisi hassusti: kaikki käyttäjienkien pyynnöt normirajapintoihinkin todettaisiin integration-api pyynnöiksi. Privileget toki rajaavat ettei aiempiin backend-rajapintoihin silloin tietysti pääsisi kuitenkaan käsiksi (tämä oli se asia jota Juuse mietti slackin puolella)

Tähän saattaa myös olla jokin mielekkäämpikin tapa, mutta vaikutti toistaiseksi suht yksinkertaiselta tehdä tämä näin.